src/test/java/api/support/APITests.java

```diff
@@ -513,6 +520,18 @@ protected void loanHasLoanPolicyProperties(JsonObject loan,
       loanPolicy.getJson().getString("name"));
   }
 
+  protected void loanHasLoanPolicyProperties(JsonObject loan,
+                                             JsonObject loanPolicy) {
```

Comment:
Please format this using standard indentation rather than horizontal alignment (that takes up more space and is harder to maintain)

Reply:
formatted